src/Physics/physicsImpostor.ts

```diff
@@ -244,12 +262,97 @@ export class PhysicsImpostor {
         this._physicsEngine.getPhysicsPlugin().setBodyRestitution(this, value);
     }
 
+    /**
+     * Gets the pressure
+     */
+    get pressure(): number {
+        if (!this._physicsEngine) {

```

Comment:
```suggestion
        if (!this._physicsEngine && this._physicsEngine.getPhysicsPlugin().getBodyPressure) {
```

Reply:
Presume same request applies to other specific softbody parameters. Will make the changes.

Reply:
Have made what I think are equivalent changes. Had problems getting past typescript error messages with your method.

Reply:
ok then my only changes would be to store the physicsPlugin jsut for the sake of not asking it twice

Reply:
Sorry but not sure of how to store the plugin and only ask once.

Reply:
something like that:
```
    if (!this._physicsEngine) {
            return;
        }
        const plugin = this._physicsEngine.getPhysicsPlugin();
        if (!plugin .setBodyPressure) {
            return;
        }
        plugin .setBodyPressure!(this, value);
```

Reply:
Will get this sorted

Reply:
Done